src/pages/workspaces/workspace/analysis/Analyses.js

```diff
@@ -389,6 +431,22 @@ const Analyses = _.flow(
       }
     }, [
       activeFileTransfers && activeFileTransferMessage,
+      feedbackShowing && h(FeaturePreviewFeedbackModal, {
+        onDismiss: () => setFeedbackShowing(false),
+        onSuccess: () => {
+          setFeedbackShowing(false)
+        },
+        featureName: 'JupyterLab',
+        formId: '1FAIpQLScgSqTwp6e2AaVcwkd8mcgseijUjBmRqT7DIyQNjdwz8IT-EA',
```

Comment:
This is a new form that I set up which I'll give IA team access to

Reply:
Would it be helpful to have the formIds in a constants file with a name for the form?

Or the whole form object from featureName to sourcePage?

Just thinking out loud. Not necessarily needed in this PR.